.circleci/config.yml

```diff
@@ -664,6 +676,7 @@ workflows:
             tags:
               only: /.*/
       - deploy:
+          name: deploy-test-no-refresh-db
```

Comment:
```suggestion
          name: deploy-tag-stage-acquia
```

Reply:
Jobs always run in the context of a workflow in Circle 2.1 so think its unnecesarily verbose to put the workflow name in each job name. i mean, its not in the build job's name, for example.

Reply:
Fine by me.